polus-fcs-to-csv-converter-plugin/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM labshare/polus-bfio-util:1.0.7-slim-buster
```

Comment:
I don't think the base image with `bfio` is required here. I think you can just use slim-buster with Python 3.6 for the base image.